HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -2286,8 +2289,6 @@ def self.apply_hvac_distribution(hpxml_bldg, ncfl, ncfl_ag)
       end
 
       # Calculate FractionDuctArea from DuctSurfaceArea
-      supply_ducts = hvac_distribution.ducts.select { |duct| duct.duct_type == HPXML::DuctTypeSupply }
-      return_ducts = hvac_distribution.ducts.select { |duct| duct.duct_type == HPXML::DuctTypeReturn }
```

Comment:
I don't think you can remove these lines of code. The code above potentially adds a new duct, so we need to get the updated list of supply/return ducts again.

Reply:
Good point, I'm going to add these lines back.